static/js/chart/draw_choropleth.ts

```diff
@@ -268,9 +270,12 @@ function generateLegend(
   chartWidth: number,
   chartHeight: number,
   color: d3.ScaleLinear<number, number>,
-  unit: string
+  unit: string,
+  margins?: { top: number; bottom: number }
```

Comment:
please describe the new parameters in the comment above.

Reply:
done